defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js

```diff
@@ -0,0 +1,108 @@
+const discordSecretName = 'LiquidationDiscordUrl';
+
+const axios = require('axios');
+const axiosRetry = require('axios-retry');
+
+axiosRetry(axios, {
+  retries: 3,
+  retryDelay: axiosRetry.exponentialDelay,
+});
+
+async function postToDiscord(url, message) {
+  const method = 'post';
+  const headers = {
+    'Content-Type': 'application/json',
+  };
+  const data = { content: message };
+
+  const response = await axios({
+    url,
+    method,
+    headers,
+    data,
+  });
+  return response;
+}
+
+// eslint-disable-next-line func-names
+exports.handler = async function (autotaskEvent) {
+  // ensure that the autotaskEvent Object exists
+  if (autotaskEvent === undefined) {
+    throw new Error('autotaskEvent undefined');
+  }
+
+  const { secrets } = autotaskEvent;
+  if (secrets === undefined) {
+    throw new Error('secrets undefined');
+  }
+
+  // ensure that there is a DiscordUrl secret
+  const discordUrl = secrets[discordSecretName];
+  if (discordUrl === undefined) {
+    throw new Error('discordUrl undefined');
+  }
+
+  // Ref: https://developer.mozilla.org/en-US/docs/Web/API/URL
+  function isValidUrl(string) {
+    let url;
+    try {
+      url = new URL(string);
+    } catch (_) {
+      return false;
+    }
+    return url.href;
+  }
+
+  if (isValidUrl(discordUrl) === false) {
+    throw new Error('discordUrl is not a valid URL');
+  }
+
+  // ensure that the request key exists within the autotaskEvent Object
+  const { request } = autotaskEvent;
+  if (request === undefined) {
+    throw new Error('request undefined');
+  }
+
+  // ensure that the body key exists within the request Object
+  const { body } = request;
+  if (body === undefined) {
+    throw new Error('body undefined');
+  }
+
+  // ensure that the alert key exists within the body Object
+  const { alert } = body;
+  if (alert === undefined) {
+    throw new Error('alert undefined');
+  }
+
+  // ensure that the alert key exists within the body Object
+  const { source } = body;
+  if (source === undefined) {
+    throw new Error('source undefined');
+  }
```

Comment:
Combine into 1 destructuring statement.
```suggestion
const { alert, source } = body;
if (alert === undefined) {
  throw new Error('alert undefined');
} else if (source  === undefined) {
  throw new Error('source undefined');
}
```